full/Dockerfile

```diff
@@ -27,6 +27,15 @@ RUN rustup target add thumbv6m-none-eabi --toolchain $RUST_NIGHTLY_VERSION
 # Adding rust-src component to nightly channel
 RUN rustup component add rust-src --toolchain $RUST_NIGHTLY_VERSION
 
+# Python packages building dependencies, can be removed afterwards
+RUN apk add -t python_build_deps python3-dev
+
+# Install ledgerctl, which is used by cargo ledger for loading or dumping an APDU app file.
+RUN pip3 install git+https://github.com/LedgerHQ/ledgerctl
```

Comment:
should be on the legacy image too I guess?

Reply:
Indeed ! Since we also have cargo-ledger in legacy. I'll add it.

Reply:
Could we use a tag or a specific sha1 to have a reproducible output image here?

Reply:
We can definitely use at least a sha1